platforms/shared/tests/test-client/main.cpp

```diff
@@ -23,6 +23,8 @@ using namespace cv;
 
 namespace prc = prism::connect;
 
+typedef boost::chrono::system_clock::time_point time_point;
```

Comment:
We all have different opinion on this. I do not think it is super critical how exactly this particular type is named. Unless somebody else will join and create majority on selecting decision, you can merge current state of it.